.xperior/testds/motr-single_tests.yaml

```diff
@@ -46,386 +46,387 @@ Tests:
 #    polltime : 30
 #    builddir : /sbin/
 
-  - id       : 02system-tests
-    script   : m0t1fs_test.sh
-    dir      : src/m0t1fs/linux_kernel/st/
+  - id       : 11m0t1fs
```

Comment:
Can we make all the test cases ordered?

Personally, I think it is good to have all of them alphabetically ordered.

Reply:
Updated